src/forms/repositorySpec.js

```diff
@@ -1,64 +1,130 @@
+            expectedFormsArguments = function(id) {             
+                return [
+                    id,
+                    hutId = jasmine.any(String),
+                    {
+                        createdAt: jasmine.any(String),
+                        updatedAt: jasmine.any(String),
+                        ...formConfigs
+                    }
+                ]
+            }
+        })
 
-        expect(actualError).not.toBe(null);
-    });
+        it("creates a form and returns the form's id", async function() {
+            let formsRepository = new FormsRepository(db);
+            let id;
 
-    it("succeeds if a form exists with the provided id", async function() {
-        let foundForm =  {
-            rows: [
-                {
-                    id: "some id",
-                    hutId: "9bdf21e7-52b8-4529-991b-5f2df9de0323",
-                    data: {
-                        showOrgType: false,
-                        showBankDetails: false,
-                        showEan: false,
-                        showCleaningToggle: false,
-                        defaultCleaningInclude: false,
-                        showArrivalTime: false,
-                        showDepartureTime: false,
-                        stdArrivalTime: false,
-                        stdDepartureTime: false,
-                        stdInformation: ""
-                    }
-                }
-            ]
-        };
+            let actualError = null;
+            try {
+                id = await formsRepository.create(hutId, formConfigs);
+            } catch(error) {
+                actualError = error; 
+            }
 
-        let db = {
-            query: jasmine.createSpy("db.query").and.callFake(async () => foundForm)
-        };
+            expect(actualError).toBe(null);
+            expect(db.query).toHaveBeenCalledWith(expectedFormsQuery,expectedFormsArguments(id));
+        });
 
-        let formsRepository = new FormsRepository(db);
-        let form;
+        it("fails if postgres throws an error while inserting a new form", async function() {
+            mockFailOnQuery(db, "INSERT INTO forms");
+            let formsRepository = new FormsRepository(db);
 
-        let actualError = null;
-        try {
-            form = await formsRepository.find("9bdf21e7-52b8-4529-991b-5f2df9de0323");
-        } catch(error) {
-            actualError = error;
-        }
+            let actualError = null;
+            try {
+                await formsRepository.create(hutId, formConfigs);
+            } catch(error) {
+                actualError = error; 
+            }
 
-        expect(actualError).toBe(null);
-        expect(form).not.toEqual(undefined);
-    });
+            expect(actualError).not.toBe(null); 
+            expect(db.query).toHaveBeenCalledWith(expectedFormsQuery,expectedFormsArguments(jasmine.any(String)));
+        });
+    })
+    
+    describe("findForm function", function() {
```

Comment:
`"find function"` should be sufficient: we are already in the forms repository test (inside a `describe("forms repository", ...`-block).